src/System.Net.Security/tests/FunctionalTests/SslStreamAlpnTests.cs

```diff
@@ -180,6 +189,38 @@ public async Task SslStream_StreamToStream_Alpn_NonMatchingProtocols_Fail()
             }
         }
 
+        [OuterLoop("Uses external server")]
+        [PlatformSpecific(TestPlatforms.OSX)]
```

Comment:
This test doesn't seem to be macOS specific, why not have it run for all platforms?